spa/package.json

```diff
@@ -2,7 +2,10 @@
   "name": "spa-core-resources",
   "license": "MPL-2.0",
   "scripts": {
-    "build": "webpack --mode production"
+    "build": "webpack --mode production",
+    "webpack-dev": "webpack --mode development --watch",
+    "browsersync": "./browsersync.js",
```

Comment:
I don't see this file - maybe I am missing something.

Reply:
Whoops! Forgot to add it. Sorry I didn't self-review. Fixed.